misc-tools/dj_make_chroot.in

```diff
@@ -67,6 +67,9 @@ Options:
   -i <debs>    List of extra package names to install (comma separated).
   -r <debs>    List of extra package names to remove (comma separated).
   -l <debs>    List of local package files to install (comma separated).
+  -s <lists>   List of apt repository .list files that exist outside the chroot 
+               to add to the chroot (comma separated). Signing keys for the
+               repository will be imported if they exist as <filename>.gpg or <filename>.arm 
```

Comment:
Does the `.arm` refer to ASCII armoured files? The standard extension for these is `.asc`, I'd use that.

Reply:
gpg --enarmor gives me an .arm instead of .asc so thats the reason why its there but its easy to just check for all 3.